pipelines/full-test-compression.yml

```diff
@@ -42,5 +42,15 @@ stages:
 
     - script: |
         cd test/algo
+        pwd
+        rm -rf ../mmdetection
+        git clone https://github.com/open-mmlab/mmdetection.git ../mmdetection
```

Comment:
You might use [Azure pipeline repo checkout syntax](https://learn.microsoft.com/en-us/azure/devops/pipelines/repos/multi-repo-checkout?view=azure-devops) to checkout a specific version of mmdetection. Better to avoid using master here because master can be potentially unstable.